rageval/metrics/_answer_rouge.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+
+from typing import List, Any, Callable, Union
+from rouge_score import rouge_scorer
+
+from datasets import Dataset
+from dataclasses import dataclass
+
+from rageval.metrics import Metric, add_attribute
+
+
+@dataclass
+@add_attribute('mtype', 'AnswerCorrectness')
+class AnswerRouge(Metric):
```

Comment:
Maybe, this metric name should be "AnswerRougeCorrectness". In this way, we can add "AnswerEMCorrectness", "AnswerNLICorrectness". Currently, the "AnswerNLICorrectness" is equal to the "AnswerClaimRecalll" as implemented in file:_answer_claim_recall.

Reply:
So we might have "AnswerEMCorrectness" and "AnswerEMGroundness" that implement Exact Match separately in AnswerCorrectness and AnswerGroundness, rather than implementing EM in a single class?